projects/igniteui-angular/src/lib/test-utils/grid-base-components.spec.ts

```diff
@@ -210,8 +211,8 @@ export class ColumnPinningWithTemplateTestComponent extends ColumnPinningTestCom
 
 @Component({
     template: `<div>
-    <igx-column-actions igxColumnPinning [grid]="grid" *ngIf="showInline"></igx-column-actions>
-    ${ GridTemplateStrings.declareGrid(` #grid [height]="height" `, ``, ColumnDefinitions.contactInfoGroupableColumns) }
+    <igx-column-actions igxColumnPinning [columns]="grid.columns" *ngIf="showInline"></igx-column-actions>
```

Comment:
this is incorrectly merged. We moved away from using a columns input as that results in expressionchangedafterithasbeenchecked exceptions. The columns input is deprecated in favor of the grid input